aggregator/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/DailyEmailAggregationJob.java

```diff
@@ -38,19 +38,22 @@
     @Channel("aggregation")
     Emitter<String> emitter;
 
-    public void processDailyEmail(Instant scheduledFireTime) {
-        List<AggregationCommand> aggregationCommands = processAggregateEmails(scheduledFireTime);
+    public void processDailyEmail() {
+        List<AggregationCommand> aggregationCommands = processAggregateEmails();
```

Comment:
This method and the one persisting the `LocalDateTime` into the database must use the same instance:
```suggestion
        LocalDateTime now = LocalDateTime.now(ZoneOffset.UTC);
        List<AggregationCommand> aggregationCommands = processAggregateEmails(now);
```